string mission_id
string root_task
string criteria
---
bool done